recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
+
+source:
+    git_url: https://github.com/PivotalRD/libhdfs3.git
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
+  skip: True  # [not linux]
+
+requirements:
+    build:
+        - protobuf
+        - libgsasl
+        - libntlm
+        - libxml2
+        - krb5
+        - libuuid
+        - boost ==1.60
+        - cmake
+    run:
+        - protobuf
+        - libgsasl
+        - libntlm
+        - libxml2
+        - krb5
+        - libuuid
+        - boost ==1.60
+
+test:
+  commands:
+    - test -f $PREFIX/include/hdfs/hdfs.h
+    - test -f $PREFIX/lib/libhdfs3.a
+    - test -f $PREFIX/lib/libhdfs3.so
+    - test -f $PREFIX/lib/libhdfs3.so.1
+    - test -f $PREFIX/lib/pkgconfig/libhdfs3.pc
```

Comment:
This library generates 11 header files in `include/hdfs/` and a library file `lib/libhdfs3.so.2.2.31`. Would others suggest that I include all of the output files (including soft links like `libhdfs3.so`) in these tests?